src/Form/Layout.php

```diff
@@ -129,115 +129,132 @@ protected function recursiveRender(): void
         $labeledGroup = $this->inputTemplate->cloneRegion('LabeledGroup');
         $noLabelGroup = $this->inputTemplate->cloneRegion('NoLabelGroup');
 
-        $this->template->del('Content');
+        $this->template->del('Content'); // TODO Content is empty in default template anyway. Needed?
```

Comment:
There are mayn changes in this PR.

Please update the commits to represent clear history of the changes. When moving a larger chunk of code somewhere, do it in a separate commit, so the real changes can be reviewed easily.

Reply:
Hi, I do not 100% get what you expect - lets have a quick ckat about this?

Reply:
I need to review moved code for corectness and then the individual changes.

So create another git branch based on develop. Then move the code into the separate methods as you did /wo any additonal changes. Then apply changes you propose in 1 or more separate commit per change. Then force push.

Reply:
understood, will do.